src/components/Comments.jsx

```diff
@@ -41,11 +41,32 @@ function Comments(props) {
         let newComment = result.data.comment[0];
         setLoaded(true);
         setComments((prev) => [newComment, ...prev]);
-        setSuccessMessage(
+        setSuccessPostMessage(
           "Thanks for your comments - your post has been added!"
         );
         setTimeout(() => {
-          setSuccessMessage(null);
+          setSuccessPostMessage(null);
+        }, 3000);
+      })
+      .catch((error) => {
+        setError(error);
+      });
+  };
+
+  //deleting a comment
+  const handleDelete = (event, comment_id) => {
```

Comment:
You have event as an argument but don't use it in the function. Please remove if not needed